### PR TITLE
Create `ListingMediaFile` entity

### LINKED ISSUE #302: Create `ListingMediaFile` enitity
### Description
Create ListingMediaFile entity to store references to listing image files. Initialize seed data.

### Requirements
- Seed data extensions and interception issue should be fixed.
- DeletedByUserId property of Listing should be nullable. Configuration and migration should be updated.
- ListingMediaFile should be convertible to imageUrl.

### Deliverables
- `ListingMediaFile` entity
- ListingImages seed data

### PR DESCRIPTION
**Pull request by Ilxom-22**
Closes #302

### CODE REVIEW

**Review by Abdurasulovich — COMMENTED** [review]
Done

File: src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Listings/Services/ListingService.cs [hunk 1]
```diff
@@ -17,9 +17,16 @@ public class ListingService(
...
+        FilterPagination filterPagination,
         bool asNoTracking = false)
-        => listingRepository.Get(predicate, asNoTracking);
+    {
+        return listingRepository.Get(asNoTracking: asNoTracking)
+            .Include(listing => listing.ImagesStorageFile
+                .OrderBy(image => image.OrderNumber))
+            .ThenInclude(media => media.StorageFile)
+            .Skip((int)((filterPagination.PageToken - 1) * filterPagination.PageSize))
+            .Take((int)filterPagination.PageSize);
```
Comment (sulton-max) [inline src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Listings/Services/ListingService.cs:28]: You can use apply pagination extension

**Review by sulton-max — APPROVED** [review]
Looks good to me

File: src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Listings/Services/ListingService.cs (hunk 1 above)
Comment (Ilxom-22) [reply src/Training.AirBnb.Clone.Backend/AirBnB.Infrastructure/Listings/Services/ListingService.cs:28]: Apply pagination extension method is only available with query specifications for now. I think it should have been fixed in #287 issue.

**Review by sulton-max — APPROVED** [review]

**Pull request merged**